connector_helpscout/models/helpscout_folder/importer.py

```diff
@@ -3,10 +3,7 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
 from odoo.addons.component.core import Component
-from odoo.addons.connector.components.mapper import (
-    mapping,
-    only_create,
-)
+from odoo.addons.connector.components.mapper import mapping, only_create
```

Comment:
It's better to format [like this](https://github.com/LasLabs/odoo-connector-carepoint/blob/release/10.0/connector_carepoint/models/address_organization.py#L8) instead